docker/cortx-deploy/cortx-control/third-party-rpms.txt

```diff
@@ -1,11 +1,12 @@
 crontabs  #cortx-utils
 java-1.8.0-openjdk-headless
-jq #cortx-prvsnr
+jq #cortx-prvsnr #cortx-hare
 which #cortx-motr
 libfabric-1.11.2 #cortx-motr
 python3-ldap #cortx-utils
 sshpass
 sudo
-python3-dbus 
-python3-m2crypto 
-python3-psutil
\ No newline at end of file
+python3-dbus
```

Comment:
I like how some lines include a comment showing which repo needs each 3rd party software. Can you add that to these?

Reply:
@johnbent, once this PR get merged, we are going to take care of it as another PR.
https://github.com/Seagate/cortx-re/blob/b2a125c0f8ebf3b283930a4d1bdd35ca3124c85e/docker/cortx-deploy/third-party-rpms.txt